src/sst/core/main.cc

```diff
@@ -215,7 +214,7 @@ doGraphOutput(SST::Config* cfg, ConfigGraph* graph, const RankInfo& myRank, cons
         graphOutputs[i]->generate(cfg, myGraph);
         delete graphOutputs[i];
     }
-    // if ( cfg->parallel_output ) delete myGraph;
+    // if ( cfg->parallel_output() ) delete myGraph;
```

Comment:
Commented out code needs to stay?

Reply:
Maybe.  But if so it needs to be uncommented.  I need to look and see what happened here.  It may be that the code commented out here and above is needed to get better outputs when parallel output is enabled.

Reply:
Uncommented the code as it was indeed needed to reduce the size of the output files.